File: LENTABP10/src/main/java/com/lenta/lentabp10/rest/DataModels/BaseSapRequest.java
package com.lenta.lentabp10.rest.DataModels;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

public class BaseSapRequest {

    @JsonProperty("format")
    @Getter private String format;

    @JsonProperty("sap-client")
    @Getter private String sapClient;

    public BaseSapRequest(String format, String sapClient) {
        this.format = format;
        this.sapClient = sapClient;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/rest/dataModels/BaseSapRequest.kt
package com.lenta.bp10.rest.dataModels

//import com.fasterxml.jackson.annotation.SerializedName
import com.google.gson.annotations.SerializedName

open class BaseSapRequest(
        @SerializedName("format")
        val format: String,
        @SerializedName("sap-client")
        var sapClient: String?)